api/api/settings/__init__.py

```diff
@@ -29,7 +25,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = list(os.environ.get("DJANGO_ALLOWED_HOSTS", "").split(" "))
+ALLOWED_HOSTS = ["*"]
```

Comment:
why?

Reply:
This is dev, we still have this on prod. This makes the API work locally even without an `.env`